store/store.go

```diff
@@ -146,6 +152,13 @@ func myappsURL() string {
 	return "https://myapps.developer.ubuntu.com/api/2.0"
 }
 
+func scaURL() string {
+	if useStagingStore() {
```

Comment:
@chipaca well it doesn't make sense to use just one staging service, maybe we should change the env var too? or support both the old env var and new one